lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -823,15 +839,75 @@ def upload_reads(self, ctx, params):
         self.log('Starting upload reads, parsing args')
```

Comment:
I guess I need to improve my python chops :)  yes, looking into it, del looks like the canonical way to remove a variable from the function scope.  I at first thought it did something more... I could have lived with this pep8 warning, but I stand corrected- looks fine.